### ISSUE TITLE
Latet version confirmation

### ISSUE THREAD

**Issue by dhawan1997** [AerialScreensaver/Aerial#83]
My Aerial Companion has been 1.5.2 for a long time, just wanna make sure that this is the latest version considering I'm running Sequoia 15.0. Just making sure I'm not using an old version. Thanks for your help!

**Comment by glouel** [comment]
That's correct yep ! And it still works in Sequoia, I'm working on V4 which will probably unify Companion and Aerial, but not fully ready yet.

**Comment by dhawan1997** [comment]
Wow thank you for your kind and swift response. Appreciate your kind guidance. Look forward to te update whenever it comes out. Would it be automatic or within app or manually to be checked?

**Comment by glouel** [comment]
No worries. You should get a notification in 1.5.2 when it comes out ! It will look a bit different than usual, but you won't be able to miss it hopefully !

**Comment by dhawan1997** [comment]
Oh alright. Thanks a lot for your hard work!!!!

**Comment by glouel** [comment]
https://github.com/AerialScreensaver/AerialCompanion/releases/tag/v3.9.9a2 unified versions are here, feel free to check but be aware it's an early version.
